tools/autograd/gen_python_functions.py

```diff
@@ -159,7 +159,6 @@
     "fill.Scalar",  # only used by the functionalization pass
     "lift.*",
     "normal_functional",  # only used by the functionalization pas
-    "_nested_view_from_buffer",  # View only version of _nested_from_buffer. This will force users to only use the "safe" version.
```

Comment:
What about using the "safe" version? :p

Reply:
I'm not entirely clear on this "safe" version comment (cc @drisspg). AFAIK this is the only way to construct an NT directly from its parts without copying, and `_nested_from_buffer` doesn't exist despite what the comment says :p

Imo we should either expose this to python or provide a proper public API for this functionality, and the latter would take design work that isn't relevant to our current focus: the subclass version of NT.

Reply:
Discussed offline with Driss: we're okay opening this private API up for now